README.md

```diff
@@ -19,7 +19,8 @@ Supports Windows, MacOS and Linux.
 - Inject Custom apps
 - Remove bloated components to improve performance
 
-#### [Installation](https://spicetify.app/docs/getting-started/installation)
+#### [Installation](https://spicetify.app/docs/getting-started/simple-installation)
+#### [Advanced Installation](https://spicetify.app/docs/getting-started/advanced-installation)
```

Comment:
I'd rather just have a referral to the `simple-installation` actually. Can you remove the advanced installation link?